lib/doctor/check-files-permission.js

```diff
@@ -0,0 +1,38 @@
+var fs = require('fs')
+var path = require('path')
+var fileCompletion = require('../utils/completion/file-completion.js')
+var getUid = require('uid-number')
+var npm = require('../npm.js')
+
+function checkFilesPermission (root, mask, cb) {
+  var accessible = true
+  if (process.platform === 'win32') return cb(null, !accessible)
```

Comment:
This should probably be calling back with `true` shouldn't it? We don't want it to mark windows as always failing right?